### ISSUE TITLE
Disadvantages of the direct use of maps

### ISSUE THREAD

**Issue by Mr03434** [maker56/UltimateSurvivalGames#16]
I used the UltimateSurvivalGames-Plugin for a minecraft-server that I'm working on. I noticed some problems, some of them influence the game in a very negative way. 
That is because for example, if a person dies, his or her items still are existent in the next game, so another player could collect them. Another problem is, that if you put a special item into a dropper, (something like a joke, or an enchanted weapon), of course you do it because you wanted to be found by players, not only once but several times. Actually, it should be possible every round. But items which are put into a vessel do not exist after beeing taken once. 
Examples for "vessels" are droppers, ovens, hoppers, dispensers.
Those problems could be solved by one measure, which is to copy the original map, so that the players do not play in the original map but in the copy. After the game, the copy gets deleted and a new copy prepared. This way, no matter what happens in the game, caused by players, it will have no consequences for the next game, because the old arena actually doesn't exist any longer. And the original map only changes if it should, i.e. when admins built something new in it.
The plugins of the most servers, at least the bigger once, work after this principe.

Another problem, which is not really caused by the direct use of maps (sorry, but i think it is less effort for both of us, if i don't write a new issue) is the access of every player to dispensers. This is, in contrast to all the other examples of vessels i named above, a problem because they often are used for traps. And of course, traps should work not once but again and again, if another player gets lost in a creepy cave or something like this. As a consequence, there are many arrows or potion in one dispenser. If the victim now survives, it can just take those stacks of arrows or potions and use them in a fight, what is very overpowered. And it is very difficult, in many cases inpossible, to built a trap in which the dispenser is out of reach. 
That's why it is more intelligent to reject the access.

If those problems are solved, i don't know what could make this plugin better, and it already is an increndible peace of work, no doupt. I apologize for any misstake in grammar, spelling or style, you could say i am still a child (16 years) and my first language is not english.
But anyway, i wish you a nice day and thanks alot for reading my issues!

**Comment by marius-meissner** [comment]
I noticed also some of the mentioned topics. Would be nice if we would get a statement of the main developer.
